test/box-luatest/gh_11929_mvcc_failing_index_count_limit_assertion_test.lua

```diff
@@ -0,0 +1,62 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group('gh-11929-mvcc-failing-index-count-limit-assertion')
+--
+-- gh-11929: mvcc failing index count limit assertion
+--
+
+g.before_all(function()
+    t.tarantool.skip_if_not_debug()
```

Comment:
This test does not rely on error injections, so I don't see why it should be skipped.